src/css/page_sections/_SecondaryContentBanner.scss

```diff
@@ -4,8 +4,10 @@
   background-color: $coa-color-light-purple;
   color: $coa-color-dark-purple;
   p {
+    max-width: 100% !important; // TODO: talk to simi about coa-main
```

Comment:
OH NO! let's talk about why you need this.
The 545px max-width was set for pages with content from the CMS -- so the content/text wouldn't extend beyond a readable width. 

Could this p be a div instead? Then we wouldn't need the max-width declaration. Or maybe the mocks have changed enough that we should probably just toss the existing css, and start over instead of trying to fight with it with !important.

Reply:
Needed to do this for the banner on the homepage...

<img width="1440" alt="screen shot 2018-02-26 at 10 28 09 pm" src="https://user-images.githubusercontent.com/5697474/36710571-759b3c00-1b44-11e8-88a0-4e6be6c1e0c6.png">

If it was intended for CMS content pages, could we apply those rules with a new class like `.coa-content` instead of applying it across the board? down to talk about this more IRL.